src/covid_model_seiir_pipeline/pipeline/forecasting/data.py

```diff
@@ -71,21 +71,16 @@ def load_location_ids(self) -> List[int]:
     def load_regression_coefficients(self, draw_id: int) -> pd.DataFrame:
         return self._get_regression_data_interface().load_regression_coefficients(draw_id=draw_id)
 
-    def load_transition_date(self, draw_id: int) -> pd.Series:
+    def load_transition_date(self, draw_id: int) -> pd.MultiIndex:
         dates_df = self._get_regression_data_interface().load_date_file(draw_id=draw_id)
         dates_df['end_date'] = pd.to_datetime(dates_df['end_date'])
-        transition_date = dates_df.set_index('location_id').sort_index()['end_date'].rename('date')
-        return transition_date
+        return dates_df['end_date'].rename('date')
```

Comment:
Is a DataFrame a MultiIndex? I thought the latter was a type of index but fundamentally more like a Series than a DataFrame

or is dates_df not really a DataFrame?

Reply:
Swapped to index and updated typing, then I realized swapping to an index was a bad idea and reverted, but did not fix the typing.